Header header
float64[] joints_rad
float32 theta_4_rad
float32[] soft_arm_pressure
float32 extrusion_length_cm
bool gripper_status
